tests/antithesis/config/docker-compose.yaml

```diff
@@ -0,0 +1,51 @@
+services:
```

Comment:
Please add a readme to antithesis directory with instruction how to run it locally.